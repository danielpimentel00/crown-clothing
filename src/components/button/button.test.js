import { shallow } from "enzyme";
import Button from "./button.component";

it("should render custom button component", () => {
  expect(shallow(<Button />)).toMatchSnapshot();
});
