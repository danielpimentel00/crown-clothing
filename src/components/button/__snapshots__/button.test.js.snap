// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render custom button component 1`] = `ShallowWrapper {}`;
